### DISCUSSION TITLE
Faster relational interfacing

### DISCUSSION THREAD

**Discussion by MAAliy** [TriliumNext/Trilium#2903] [Ideas]
it's too slow, can you add a model to be like [this](https://www.youtube.com/watch?v=sVOltbzco9Y)? 
End result
![49009507-1693ea80-f1b4-11e8-805a-2ada3eeb089a](https://user-images.githubusercontent.com/71061448/172725396-c4150bc9-89a8-478c-b047-ffc9f5723820.gif)
Page: https://github.com/cotoami/cotoami/releases

**Comment by zadam** [comment]
Hi, Trilium has similar feature called "note map":

![image](https://user-images.githubusercontent.com/617641/172925536-70499502-e7c0-4be8-b817-5cfeb2b6c481.png)

Or did you mean something more specific?

**Reply by MAAliy** [reply]
This is great, why is there no back option while using it? You've created an exceptional software, please upload a workflow video. Doesnt need any speech, just how to make a map around a random topic you've found interest in. I want to use it as replacement for docear, for example a paper that has many references, several passages that require more references, references of refrences, and so on. How does one go about creating a concept map of such on trillium? 
In essence combining these two tools: https://nodebook.io/nodebook/725/t/x=248.59&y=828.62&k=0.66
https://topictrails.com/app/ontology
Docear: https://docear.org/support/user-manual/

Interfacing, creating new connection and nodes from the map would be greatly appreciated.
